### PR TITLE
Cvsl-1750-other-tab-email-cvl-support-batch-nomsIds

### PR DESCRIPTION
**Pull request by nishanthmahasamudram**

### CODE REVIEW

File: detekt-baseline.xml [hunk 1]
```diff
@@ -56,6 +56,7 @@
     <ID>MagicNumber:ExclusionZoneService.kt$ExclusionZoneService$500</ID>
     <ID>MagicNumber:LicenceActivationService.kt$LicenceActivationService$4</ID>
     <ID>MagicNumber:LicenceService.kt$LicenceService$14L</ID>
+    <ID>MagicNumber:NotifyAttentionNeededLicencesService.kt$NotifyAttentionNeededLicencesService$500</ID>
```
Comment (andrewrlee) [inline detekt-baseline.xml:59]: you could remove this warning by extract it into a nicely named constant?

**Review by andrewrlee — APPROVED** [review]
LGTM

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/NotifyAttentionNeededLicencesService.kt [hunk 2]
```diff
@@ -31,7 +32,10 @@ class NotifyAttentionNeededLicencesService(
       return
     }
 
-    val prisoners = prisonerSearchApiClient.searchPrisonersByNomisIds(attentionNeededLicences.map { it.nomsId!! })
+    val batchedNomsIds = attentionNeededLicences.map { it.nomsId!! }.chunked(Companion.BATCH_SIZE)
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/NotifyAttentionNeededLicencesService.kt:35]: no need to change but you don't need to refer to this via `Companion.`

Comment (nishanthmahasamudram) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/NotifyAttentionNeededLicencesService.kt:35]: will change it in next PR.

**Pull request merged**